File: cgi-bin/web_scripts/id.pl
#!/usr/bin/perl -w
use strict;
use warnings;

use FindBin;

BEGIN { require File::Spec->catfile( $FindBin::Bin, 'requireLibrary.pl' ); }
use PipelineMiRNA::WebTemplate;

my @css = (PipelineMiRNA::WebTemplate->get_server_css_file(), PipelineMiRNA::WebTemplate->get_css_file());
my @js  = (PipelineMiRNA::WebTemplate->get_js_file());

my $page = <<"END_TXT";
<div class="main">
  <h2>Retrieve result with an ID</h2>
  <div class="forms">
    <form method="post" action="./resultsWithID.pl">
      <p>The ID remains valid 15 days after sequence submission.</p>
      <label for='run_id'><b>Enter the ID:</b></label>
        <input type="text" name="run_id" id='run_id' size="20">
        <input type="hidden" name="command" value="result">
        <input type="submit" value="Go">
      </p>
    </form>
  </div> <!-- form -->
</div>
END_TXT

my $html = PipelineMiRNA::WebTemplate::get_HTML_page_for_content($page, \@css, \@js);

print <<"DATA" or die("Error when displaying HTML: $!");
Content-type: text/html

$html
DATA
###End###

File: html/id.php
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
    <head>
        <link type='text/css' rel='stylesheet' href='./style/mirkwood.css' />
        <link type='text/css' rel='stylesheet' href='./style/script.css' />
        <link type='text/css' rel='stylesheet' href='./style/rna.css' />
        <link type='text/css' rel='stylesheet' href='../Style/css/bioinfo.css' />
        <script type="text/javascript" src="/scripts/bioinfo.js"></script>
        <script type="text/javascript" src="../../libs/jquery-1.11.3.min.js"></script>
        <script type="text/javascript" src="../../libs/jquery.history.js"></script>
        <script type="text/javascript" src="./js/header.js"></script>

        <title>miRkwood - MicroRNA identification</title>
    </head>

    <body>
        <div class="frametitle">
            <h1 id="title" onclick="loadLink('/mirkwood/index.php');">miRkwood</h1>
        </div>

        <div class="tabs" id="menu_central" style="display:inline-block">
            <?php include("./static/header_menu.txt") ?>
        </div>
        <div id="arborescence"></div>

        <div id="main">

            <div id="mirkwood_title">Retrieve result with an ID</div>
            <br/>

            <div class="forms">
                <form method="post" action="/cgi-bin/mirkwood/web_scripts/retrieve_job_with_ID.pl">
                    <p>The ID remains valid 15 days after sequence submission.</p>
                    <p>
                        <label for='jobId'><b>Enter the ID:</b></label>
                        <input type="text" name="jobId" id='jobId' size="20"/>
                        <input type="hidden" name="command" value="result"/>
                        <input type="submit" value="Go"/>
                    </p>
                </form>
            </div> <!-- form -->

        </div> <!-- main -->

        <?php require("/bio1/www/html/lib.inc")?>
        <?php footer("miRkwood","miRkwood", "mirkwood@univ-lille1.fr","2013"); ?>

    </body>

</html>
